tests/test_read_student_file.py

```diff
@@ -0,0 +1,12 @@
+""" Testing read_student_file """
+
+# pylint: disable=redefined-outer-name
+import pytest
+
+from gatorgrouper.utils import read_student_file
+
+
+def test_read_student_file(generatcsv):
+    """ Test read_student_file """
+    expectedoutput = [["delgrecoj", True, True, False, True]]
+    assert read_student_file.read_student_file() == expectedoutput
```

Comment:
How does this pass? `read_student_file` expects a filepath to be given as a parameter -- that is what `generatcsv` is for (also a misspelling in that).